GenericBot/GenericBot.cs

```diff
@@ -58,6 +60,20 @@ static void Main(string[] args)
                 Logger.LogGenericMessage($"Build {File.ReadAllText("version.txt").Trim()}");
                 BuildNumber = File.ReadAllText("version.txt").Trim();
             }
+
+            if (File.Exists("emoji_replacements.json"))
```

Comment:
Please include this dictionary in the Entities.GuildConfig class and refactor code appropriately.